lib/opal/builder.rb

```diff
@@ -49,23 +50,21 @@ def self.register_processor(processor, processor_extensions)
       processor_extensions.each { |ext| extensions << ext }
     end
 
-
-
```

Comment:
Two spaces may mean a beginning of a section, but I'd prefer having a comment that explicitly states it:
``` ruby
def top_section_method
end

# Bottom section

def bottom_section_method
end
```

I guess the reason why rubocop complains about it is that two spaces can be just two spaces 😄

Reply:
> I guess the reason why rubocop complains about it is that two spaces can be just two spaces 😄

sure and of course being a robot it can't discern if there's a meaning in it, but I agree about _also_ using comments